BLSQ/iaso
# POLIO-1125-Backend multi round RA and budget

Added a Model named Process to allows budget to be multiround. 
A process is linked to round(s) that are linked to a campaign. 

Related JIRA tickets : POLIO-1125

## Self proofreading checklist


## How to test
There is no interface. you must go to /api/polio/budgetprocesses/ or django admin and create a process after creating a budget 
You can also check the response from the budget and there should be the proccess info in the json.
/api/polio/budget/
The setup is quite cumbersome. 

Explain how to test your PR.
If a specific config is required explain it here (dataset, account, profile, ...)

## Print screen / video

![image](https://github.com/BLSQ/iaso/assets/45785235/c14d4cd9-8a47-4881-9693-30ebf5952c05)
## Notes

Things that the reviewers should know: known bugs that are out of the scope of the PR, other trade-offs that were made.
If the PR depends on a PR in bluesquare-components, or merges into another PR (i.o. main), it should also be mentioned here
